components/eam/src/chemistry/utils/modal_aero_deposition.F90

```diff
@@ -473,6 +540,35 @@ subroutine set_srf_drydep(aerdepdryis, aerdepdrycw, cam_out)
       ! distributions of MAM7 fine dust and coarse dust shown in Table
       ! 1 of Liu et al (2012, doi:10.5194/gmd-5-709-2012).  In MAM3,
       ! accumulation-mode dust is assumed to resemble fine dust
+#if ( defined MOSAIC_SPECIES )
```

Comment:
@mingxuanwupnnl , Please double check if the negative sign is needed for the dust dry depo terms. The wet depo and dry depo are also inconsistent in the sign convention.

Reply:
Thank you. Just corrected them. Should not have the negative sign fro dry dep.